tools/dev/k3d/environments/helm-cluster/main.jsonnet

```diff
@@ -0,0 +1,183 @@
+               + k.apps.v1.deployment.hostVolumeMount(
+                 name='enterprise-logs-app',
+                 hostPath='/var/lib/grafana/plugins/grafana-enterprise-logs-app/dist',
+                 path='/grafana-enterprise-logs-app',
+                 volumeMixin=k.core.v1.volume.hostPath.withType('Directory')
+               )
+               + k.apps.v1.deployment.emptyVolumeMount('grafana-var', '/var/lib/grafana')
+               + k.apps.v1.deployment.emptyVolumeMount('grafana-plugins', '/etc/grafana/provisioning/plugins')
+               + k.apps.v1.deployment.spec.template.spec.withInitContainersMixin([
+                 container.new('startup', 'alpine:latest') +
+                 container.withCommand([
+                   '/bin/sh',
+                   '-euc',
+                   |||
+                     mkdir -p /var/lib/grafana/plugins
+                     cp -r /grafana-enterprise-logs-app /var/lib/grafana/plugins/grafana-enterprise-logs-app
+                     chown -R 472:472 /var/lib/grafana/plugins
+
+                     cat > /etc/grafana/provisioning/plugins/enterprise-logs.yaml <<EOF
+                     apiVersion: 1
+                     apps:
+                       - type: grafana-enterprise-logs-app
+                         jsonData:
+                           backendUrl: %s
+                           base64EncodedAccessTokenSet: true
+                         secureJsonData:
+                           base64EncodedAccessToken: "$$(echo -n ":$$GEL_ADMIN_TOKEN" | base64 | tr -d '[:space:]')"
+                     EOF
+                   ||| % lokiGatewayUrl,
+                 ]) +
+                 container.withVolumeMounts([
+                   k.core.v1.volumeMount.new('enterprise-logs-app', '/grafana-enterprise-logs-app', false),
+                   k.core.v1.volumeMount.new('grafana-var', '/var/lib/grafana', false),
+                   k.core.v1.volumeMount.new('grafana-plugins', '/etc/grafana/provisioning/plugins', false),
+                 ]) +
+                 container.withImagePullPolicy('IfNotPresent') +
+                 container.mixin.securityContext.withPrivileged(true) +
+                 container.mixin.securityContext.withRunAsUser(0) +
+                 container.mixin.withEnv([
+                   envVar.fromSecretRef('GEL_ADMIN_TOKEN', adminTokenSecret, 'token'),
+                 ]),
+               ]) + k.apps.v1.deployment.mapContainers(
+                 function(c) c {
+                   env+: [
+                     envVar.new('GF_PLUGINS_ALLOW_LOADING_UNSIGNED_PLUGINS', 'grafana-enterprise-logs-app'),
+                     envVar.fromSecretRef('PROVISIONED_TENANT_TOKEN', '%s-%s' % [provisionedSecretPrefix, tenant], 'token-read'),
+                   ],
+                 }
+               ),
+           },
+
+  minioNamespace: k.core.v1.namespace.new('minio'),
+  minio: helm.template('minio', '../../charts/minio', {
```

Comment:
Could we have a values file instead? I'd love to document the steps without some automation.